src/cpu/aarch64/jit_uni_reorder.cpp

```diff
@@ -791,7 +791,7 @@ struct jit_uni_reorder_kernel_f32_t : public kernel_t, public jit_generator {
             // transposition on the fly
             const bool fast_return = prb_.src_scale_type != scale_type_t::MANY
                     && prb_.dst_scale_type != scale_type_t::MANY
-                    && prb_.beta == 0.f;
+                    && prb_.beta == 0.f && !prb_.req_src_zp && !prb_.req_dst_zp;
```

Comment:
Just curious, do we need `!prb_.req_dst_zp` here? In my experimentation the dst zero point wasn't causing issues but I may not have found the right shapes which would cause an issue

Reply:
@Ryo-not-rio , thanks for the review.
So this looks like it was initially ported from x86 jit_uni_reorder. When I compared assemblies of the 2 I figured this was the point at which it was behaving differently. So for now I have used the same condition used in x86. We can do further testing in future to remove the condition for `!prb_.req_dst_zp` . But for now I feel its safe to keep it. Anyway this would still fall to jit:uni but just a fast_return feature would be disabled.